Replicator/ChangesFeed.cc

```diff
@@ -102,6 +102,7 @@ namespace litecore::repl {
         try {
             _db.useLocked([&](C4Database* db) {
                 Assert(db == _checkpointer->collection()->getDatabase());
+                options.flags |= kC4RevIDGlobalForm;
```

Comment:
Instead of changing C4DocEnumerator, you can just call `DBAccess::convertVersionToAbsolute` on the revID.